modules/virt-subscribing-cli.adoc

```diff
@@ -94,3 +94,40 @@ for {VirtProductName} by running the following command:
 ----
 $ oc apply -f <file name>.yaml
 ----
+
+.Verification
+
```

Comment:
```suggestion

```

Make this first step about the subscription part of the main procedure if it triggers OLM to do necessary steps. If not, it might need to be reworded to make it clearer.